File: server/main/Card.java
package main;

import com.smartfoxserver.v2.protocol.serialization.SerializableSFSType;

public abstract class Card implements SerializableSFSType {
	
	protected Bandit belongsTo;
	
	// NEEDED FOR SERIALIZATION
	public Card() { }
	
    Bandit getBelongsTo() {
        return this.belongsTo;
    }

    boolean setBelongsTo(Bandit newObject) {
        this.belongsTo = newObject;
        return true;
    }
}

File: colt-express-jan-2021/Assets/Scripts/model/src/main/Card.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using Sfs2X;
using Sfs2X.Logging;
using Sfs2X.Util;
using Sfs2X.Core;
using Sfs2X.Entities;
using Sfs2X.Entities.Data;
using Sfs2X.Protocol.Serialization;

public abstract class Card : MonoBehaviour, SerializableSFSType {
    
    public Bandit belongsTo;
    
    Bandit getBelongsTo() {
        return this.belongsTo;
    }
    
    bool setBelongsTo(Bandit newObject) {
        this.belongsTo = newObject;
        return true;
    }
}